workflow/tools/chem_prep_dust.sh

```diff
@@ -46,7 +46,7 @@ else
       ncap2 -O -s 'albedo_drag_m_in=feff_m_in' "${DUST_INITFILE}" "${DUST_INITFILE}"
    fi   
    ncks -O -6 "${DUST_INITFILE}" "${DUST_INITFILE}"
-   timestr3=$(date +%Y-%m-%d_%H:00:00 -d "$current_day")
+#   timestr3=$(date +%Y-%m-%d_%H:00:00 -d "$current_day")
    #shellcheck disable=SC2086
-   ncap2 -O -s xtime=\"${timestr3}\" "${DUST_INITFILE}" "${DUST_INITFILE}"
+#   ncap2 -O -s xtime=\"${timestr3}\" "${DUST_INITFILE}" "${DUST_INITFILE}"
```

Comment:
Do we still need lines 49-51? if not, we can remove them.

Reply:
will do